nova/scheduler/weights/ram.py

```diff
@@ -36,3 +36,24 @@ def weight_multiplier(self):
     def _weigh_object(self, host_state, weight_properties):
         """Higher weights win.  We want spreading to be the default."""
         return host_state.free_ram_mb
+
+
+class RamResizeBigVmToSameHostWeigher(weights.BaseHostWeigher):
```

Comment:
Please create a new file for this weigher and rename it. We want all VMs to be resized on the same host, if possible, not only bigVMs. I'd prefer it to be name `PreferSameHostOnResizeWeigher`, because that's what it's supposed to do.